.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: build (${{ matrix.ruby }}
+    strategy:
+      matrix:
+        ruby: [ 3.0, 2.7, head ]
```

Comment:
Quote the 3.0, as "3.0", to avoid it becoming the Float 3.0, which turns into the String "3".